use-cases/run_jcl/README.md

```diff
@@ -0,0 +1,80 @@
+# Zowe CLI SonarQube Sample
+This sample serves as a working example of using Zowe SDK.
+
+The sample includes a means to execute and check JCL output locally via `npm` scripts and via a Jenkins pipeline.
+
+The sample will upload the JCL source file from the local file specified in [custom_properties.json](./custom_properties.json), run it on the mainframe, and download the output.
+
+## Quick-Start
+For detailed information on configuration and execution, start with [Prerequisites](#Prerequisites).
+
+1. Clone the project
+2. Configure [custom_properties.json](./custom_properties.json)
```

Comment:
This links to a non-existent file. Should this readme include a step similar to the Assembler Metal readme?
> Make a copy of `properties.json` and name it `custom_properties.json`.

Reply:
Fixed